NSubstitute.DbConnection/NSubstitute.DbConnection/Extensions/DataTableExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace NSubstitute.DbConnection.Extensions
+{
+    using System.Data;
+
+    internal static class DataTableExtensions
```

Comment:
Can you implement this more idiomatically as ` internal class DataTableCompararer : IEqualityComparer<DataTable>`?